apps/grader-host/package.json

```diff
@@ -44,11 +44,23 @@
     "@types/node": "^20.12.11",
     "@types/sinon": "^17.0.3",
     "@types/tmp": "^0.2.6",
+    "c8": "^9.1.0",
     "mocha": "^10.4.0",
     "nodemon": "^3.1.0",
     "sinon": "^17.0.1",
-    "tsx": "^4.10.2",
+    "tsx": "^4.10.4",
     "typescript": "^5.4.5",
     "typescript-cp": "^0.1.9"
+  },
+  "c8": {
+    "reporters": [
+      "html",
+      "text-summary",
+      "cobertura"
+    ],
+    "all": true,
+    "include": [
+      "src/**"
+    ]
```

Comment:
Thoughts on putting this config in `package.json` as opposed to `.c8rc.json`?

Reply:
I'm fine to have it in `package.json`. I'm kind of split on which one is easier to find when you are trying to figure out where the config is for `c8`.

Reply:
I'll leave it here for now, this is easy enough to change down the line if we want.